SpatialGDK/Source/SpatialGDK/Private/Tests/WorkerViewTest.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+
+#include "WorkerView.h"
+#include "OpList/ViewDeltaLegacyOpList.h"
+
+#define WORKERVIEW_TEST(TestName) \
+	GDK_TEST(Core, WorkerView, TestName)
+
+using namespace SpatialGDK; 
+
+namespace
+{
+	Worker_Op CreateEmptyCreateEntityResponseOp()
+	{
+		Worker_Op Op{};
+		Op.op_type = WORKER_OP_TYPE_CREATE_ENTITY_RESPONSE;
+		Op.op.create_entity_response = Worker_CreateEntityResponseOp{};
+		return Op;
+	}
+
+} // anonymous namespace
+
+WORKERVIEW_TEST(GIVEN_empty_WorkerView_WHEN_GenerateViewDelta_called_THEN_ViewDelta_returned)
```

Comment:
As this bit of the API is still in flux testing pipes explicity might not be all that helpful. This will likely be tested as part of testing the component interactions in the view delta.